src/handy.jl

```diff
@@ -326,6 +326,17 @@ function mysql_metadata(hndl::MySQLHandle)
     return MySQLMetadata(mysql_metadata(hndl.stmtptr))
 end
 
+"""
+    mysql_escape(hndl::MySqlHandle, str::AbstractString) -> String
+
+Escapes a string using `mysql_real_escape_string()`, returns the escaped string.
+"""
+function mysql_escape(hndl::MySQLHandle, str::AbstractString)
+    output = Vector{UInt8}(length(str)*2 + 1)
+    output_len = mysql_real_escape_string(hndl.mysqlptr, output, str, UInt64(length(str)))
```

Comment:
`MySQLInternalError` will call `mysql_error` providing the reason for the error. So it would be good to have that check here `output_len == -1 && throw(MySQLInternalError(hndl))` before the return.